lib/modules/home/components/slide_item.dart

```diff
@@ -0,0 +1,89 @@
+part of '../home_module.dart';
+
+class SlideItem extends StatelessWidget {
+  final SurveyInfo survey;
+  const SlideItem({
+    Key? key,
+    required this.survey,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      fit: StackFit.expand,
+      children: [
+        Bone.hidden(
+          child: Image(
+            image: NetworkImage(survey.coverImageUrl!),
+            fit: BoxFit.fill,
+          ),
+        ),
+        Align(
+          alignment: Alignment.bottomCenter,
+          child: SafeArea(
+            child: Container(
+              margin: const EdgeInsets.fromLTRB(20, 0, 20, 0),
+              child: SizedBox(
+                height: 126,
+                child: Row(
+                  children: [
+                    Expanded(
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          Bone.multiple(
+                            number: 2,
+                            variants: const [1, 0.5],
+                            child: Text(
+                              survey.title!,
+                              style: const TextStyle(
+                                color: Colors.white,
+                                fontSize: 28,
+                              ),
+                            ),
+                          ),
+                          const SizedBox(
+                            height: 16,
+                          ),
+                          Bone.multiple(
+                            number: 2,
+                            variants: const [1, 0.5],
+                            child: Text(
+                              survey.description!,
+                              style: TextStyle(
+                                color: Colors.white.withOpacity(0.7),
+                                fontSize: 17,
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                    PlatformButton(
+                      child: Bone(
+                        width: 56,
+                        height: 56,
+                        borderRadius: BorderRadius.circular(28),
+                        child: Container(
+                          decoration: BoxDecoration(
+                            borderRadius: BorderRadius.circular(28),
+                            color: Colors.white,
+                          ),
+                          width: 56,
+                          height: 56,
+                          child: Assets.images.arrowRightIcon
+                              .svg(fit: BoxFit.none),
+                        ),
```

Comment:
there is small issue with icon background, it has a rectangle opacity around the button, could you please fix? 🙏